src/aframe-streetmix-parsers.js

```diff
@@ -149,9 +150,9 @@ function createDividerVariant (variantName, positionX, clonedObjectRadius, step
   return dividerParentEl;
 }
 
-function createClonedVariants (variantName, positionX, clonedObjectRadius, step = 2.25) {
+function createClonedVariants (variantName, positionX, clonedObjectRadius, step = 2.25, rotation = '0, 0, 0') {
```

Comment:
pretty sure this should be `rotation = '0 0 0'`

Reply:
Nice catch :+1: I just added a commit that addresses this.